pulldown-cmark/src/parse.rs

```diff
@@ -1688,7 +1772,8 @@ struct LinkStackEl {
 
 #[derive(PartialEq, Clone, Debug)]
 enum LinkStackTy {
-    Link,
+    // if this is doubled up, could be a wikilink
+    Link(bool),
```

Comment:
Obsidian and Gollum both have ways to create image-style wikilinks.

- https://help.obsidian.md/Linking+notes+and+files/Embed+files
- https://github.com/gollum/gollum/wiki#linking-internal-images

I suspect this is common in most wiki tools.